apps/studio/components/interfaces/Functions/EdgeFunctionSecrets/AddNewSecretForm.tsx

```diff
@@ -224,6 +224,7 @@ const AddNewSecretForm = () => {
                     type="default"
                     className="h-[34px] mt-6"
                     icon={<MinusCircle />}
+                    disabled={fields.length <= 1}
```

Comment:
This “delete row” button is irrelevant if only one row is around. We could add a tooltip here but it feels pretty explanatory.